WeScan/Scan/CaptureSessionManager.swift

```diff
@@ -7,8 +7,11 @@
 //
 
 import Foundation
+import CoreMotion
 import AVFoundation
 
+var editImageOrientation: CGImagePropertyOrientation = .downMirrored
```

Comment:
For what reason are we defining this property as a global? Can't we solve this in a different way?

Reply:
This is to rotate the image when it's being passed to the ReviewVC, so that it'll work if the person rotates their phone after scanning a picture.

Reply:
I understand, but why do we define it as a global parameter and not just as a property within a class for example?

Reply:
The CaptureSessionManager class is initialized from the ScannerViewController, while the ReviewViewController is initialized from the EditScanViewController. I think it would be best to keep this as a global property, but if anyone knows a better way of doing this feel free to suggest it and i'll push it.

Reply:
I'm in favor of using a Singleton in this case. 
There can only be one orientation and one capture session at any given time, and they're relevant during the whole life of the scanner. 
I would use `CaptureSession.current`.

Reply:
This has finally been implemented in 9083131 using `CaptureSession.current`.